examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3153,6 +3153,10 @@ class GLTFParser {
 
 				}
 
+				// Ignore normalized since we copy from sparse
+				const prevNormalize = bufferAttribute.normalized;
```

Comment:
nit: Probably not necessary to save `prevNormalize` here, the original `normalized` variable is still in scope.

Reply:
Indeed, updated the commit